### PR TITLE
Fix cmake dependencies

### PR DESCRIPTION
**Pull request by jnbrunet**
Following SOFA's [pluginization](https://github.com/sofa-framework/sofa/issues/1527)  , this PR clean up modules dependencies.

### CODE REVIEW

File: bindings/Sofa/src/SofaPython3/Sofa/Core/CMakeLists.txt [hunk 1]
```diff
@@ -67,15 +67,15 @@ if (NOT TARGET SofaPython3::Plugin)
 endif()
 
 find_package(SofaFramework REQUIRED)
-find_package(SofaBase REQUIRED)
-find_package(SofaCommon REQUIRED)
 find_package(SofaSimulation REQUIRED)
+find_package(SofaBaseVisual REQUIRED)
```
Comment (fredroy) [inline bindings/Sofa/src/SofaPython3/Sofa/Core/CMakeLists.txt:71]: I understand the need of BaseUtils (for requiredplugin I presume!) but why SofaBaseVisual ?

Comment (jnbrunet) [reply bindings/Sofa/src/SofaPython3/Sofa/Core/CMakeLists.txt:71]: @fredroy for this:

https://github.com/sofa-framework/SofaPython3/blob/f99e60dbd82df619d99a445f227efe257f32590a/bindings/Sofa/src/SofaPython3/Sofa/Core/Binding_BaseCamera.cpp#L32

Comment (fredroy) [reply bindings/Sofa/src/SofaPython3/Sofa/Core/CMakeLists.txt:71]: A bit weird that something from a non-core module is bind into the Core module.
I guess this should be moved somewhere in the future for more consistency (but not the point of this PR)

Comment (jnbrunet) [reply bindings/Sofa/src/SofaPython3/Sofa/Core/CMakeLists.txt:71]: Absolutely

**Pull request merged**